makeup/html/stub_page.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <title>CashTan</title>
+    <link rel="stylesheet" href="shared.css">
+    <link rel="stylesheet" href="stub_page.css">
+    <link rel="stylesheet" href="animate.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script defer src="https://use.fontawesome.com/releases/v5.0.10/js/all.js"
+            integrity="sha384-slN8GvtUJGnv6ca26v8EzVaR9DC58QEwsIk9q1QXdCU8Yu8ck/tL/5szYlBbqmS+"
+            crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
+    <link rel="shortcut icon" href="images/cashtan_favicon.ico" type="image/x-icon">
+    <meta name="viewport" content="width=device-width">
+</head>
+
+<body>
+<div class="container">
+    <div class="logo animated hinge">
+        <div class="list-name">
+            <img class="logo-menu" src="images/cashtan-min.png">
+            <strong class="black">CashTan</strong>
```

Comment:
name black is bad for class. You must fix it..

Reply:
fixed